src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectsParser.java

```diff
@@ -44,4 +45,8 @@ public ProjectsParser(Gson gson) {
     public ProjectInfo parseSingleProjectInfo(JsonElement result) {
         return gson.fromJson(result, ProjectInfo.class);
     }
+
+    public String generateProjectInput(ProjectInput input) {
```

Comment:
Parser classes are used for parsing JSON -> Java objects in all other places. The other way around is done in the *Client classes at the moment. Not sure if this is the best way, but I would like to keep it consistent.